draft-ietf-rats-endorsements.md

```diff
@@ -201,16 +201,38 @@ some minimal matching policy (e.g., exact match against a singleton reference
 value).  This unfortunately complicates design as a Verifier may need
 multiple parsers for matching policies.
 
-# Endorsing Identity
-
-One type of claims that might be endorsed would be claims having to do with
-identity, such as verification keys bound to an identifier for the Attesting Environment (e.g., {{-rats-eat}} `ueid`).  While identity claims are just another
+# Key Material for Attestation
+
+A Verifier receives Evidence from an Attester that must be considered untrusted
+until verified through cryptography. Typically,
+the bottom-most Attesting Environment in an Attester will sign claims about one or more Target Environments
+with a private key that the Attesting Environment possesses and the Verifier will verify
+the resulting Evidence with a public key it possesses, called a verification key below. While this is typical,
+cryptography other than public key may also be used.
+
+Since it is not possible to establish trust in an Attester without cryptography,
+the Verifier must have access to a verification key for each Attester. Such a key
+could be provisioned directly in the Verifier, though for scalability the Verifier
+typically is provisioned with a trusted root CA certificate such that an
+Endorsement from an Endorser includes the Attester's verification key material
+in the form of a certificate that chains up to that trusted root.
```

Comment:
hmm, providing the verification key material in the form of a certificate doesn't seem to solve the scalabilty problem, it's still 1:1 with the attesters' population.

If you have the right trust anchor configured, the endorsement consists of the intermediates' certs that allow completing the path to the Attester's cert, i.e.:

```
TA -> iCA1 -> ... -> iCAn -> attesters' certs
      '-----------------'
          endorsement
```
Note that this kind of endorsement could be inlined with Evidence, e.g., in EAT (or other CWT-/JWT-based formats) using an `x5chain` header parameter.

Reply:
Scalability is solved by not storing the attester's certificate.  It still comes in an endorsement, but whether that is resolved on demand or passed along with the evidence depends on the protocol.

Reply:
> Scalability is solved by not storing the attester's certificate.

OK, then I don't understand what "Attester's verification key material" means in the following:

_"[e]ndorsement from an Endorser includes the Attester's verification key material
in the form of a certificate that chains up to that trusted root."_

Does that exclude the leaf cert?

Reply:
A certificate that includes the Attester's public key, signed by the Endorser.  I'd call that the leaf cert.  I'm saying the leaf cert need not be stored by the Verifier.  Thus the Verifier's persistent state need not be O(Attesters).

Reply:
> I'm saying the leaf cert need not be stored by the Verifier. Thus the Verifier's persistent state need not be O(Attesters).

That's the key point, but lines 217-218 are not making that entirely transparent. I suggest we explicitly state it.

Reply:
Ok, added sentence, please check.

Reply:
Thanks very much.